test/electron/fixtures/events/main/breadcrumbs/network/error.json

```diff
@@ -71,13 +71,7 @@
         {
           "errorMessage": "something bad",
           "errorClass": "ReferenceError",
-          "stacktrace": [{
-            "file": "./src/errors.js",
-            "lineNumber": 18,
-            "code": {
-              "1": "{TYPE:string}"
-            }
-          }],
+          "stacktrace": [],
```

Comment:
The stacktraces now look like this:

```
                {
                    "errorClass": "ReferenceError",
                    "errorMessage": "something bad",
                    "type": "electronnodejs",
                    "stacktrace": [
                        {
                            "file": ".webpack/main/index.js",
                            "method": "ClientRequest.notify",
                            "lineNumber": 2,
                            "columnNumber": 86370,
                            "code": {
                                "1": "/*! For license information please see index.js.LICENSE.txt */",
                                "2": "(()=>{var e={1821:e=>{e.exports=class{constructor(e,a,i,n=new Date){this.type=i,this.message=e,this.metadata=a,this.timestamp=n}toJSON(){return{type:this.type,name:this.message,timestamp:this.timestam"
                            }
                        },
                        {
                            "file": "node:events",
                            "method": "ClientRequest.emit",
                            "lineNumber": 531,
                            "columnNumber": 35
                        },
                        {
                            "file": "node:electron/js2c/browser_init",
                            "method": "SimpleURLLoaderWrapper.<anonymous>",
                            "lineNumber": 2,
                            "columnNumber": 114542
                        },
                        {
                            "file": "node:events",
                            "method": "SimpleURLLoaderWrapper.emit",
                            "lineNumber": 519,
                            "columnNumber": 28
                        }
                    ],
                    "message": "something bad"
                }
```

With the original fixture, the error is:

```
   ✖ Then the contents of an event request matches "main/breadcrumbs/network/error.json" # test/electron/features/support/steps/request-steps.js:202
       Error: expect.toContainPayload(expected) - Nearest match (of 1):
         .events.0.exceptions.0.stacktrace.0.file: Expected an exact match
           Expected: ./src/errors.js
           Received: node:events

         .events.0.exceptions.0.stacktrace.0.lineNumber: Expected an exact match
           Expected: 18
           Received: 519

         .events.0.exceptions.0.stacktrace.0.code: Expected a value but was undefined
           Expected: {
         "1": "{TYPE:string}"
       }
           Received: undefined

           at World.<anonymous> (/Users/dan.skinner/bugsnag-js/test/electron/features/support/steps/request-steps.js:212:20)
```